contrib/intel/jenkins/build.py

```diff
@@ -97,7 +102,7 @@ def log_dir(install_path):
     p = re.compile('mpi*')
 
     if (build_item == 'libfabric'):
-        build_libfabric(libfab_install_path, ofi_build_mode)
+        build_libfabric(install_path, ofi_build_mode, cluster)
```

Comment:
this needs to be libfab_install_path otherwise the three different builds will not have their own unique tags